src/Command/MakeMigrationCommand.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Command;
+
+use Doctrine\Bundle\MigrationsBundle\Command\DoctrineCommand;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Bundle\MakerBundle\Validator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\CommandNotFoundException;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+final class MakeMigrationCommand extends AbstractCommand
+{
+    protected static $defaultName = 'make:migrations';
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Creates a migration with the diff or an empty migration if there are no diffs')
```

Comment:
After some more thinking, I think we should *not* automatically create an empty migration if the diff is empty. Instead, we should show a message:

> No changes were detected to your mapping metadata.
> If you want to generate an empty migration, re-run the command with --empty

Reply:
or sth like this: `make:migration --allow-empty`, `make:migration:diff` (`make::diff-migration`) or `make:migration:empty` (`make:empty-migration`)

Reply:
That looks too much and too complicated. Simplicity is the main feature of this bundle. Thanks!